src/App.js

```diff
@@ -1,25 +1,57 @@
 import React from 'react';
 
 import './App.scss';
+import { Clock } from './components/Clock/Clock';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
+class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    name: 'Name',
```

Comment:
```suggestion
    name: `Name ${Math.round(Math.random() * 100)}`',
```
It's better to make initial value also random